src/app/login/login.component.html

```diff
@@ -12,8 +12,7 @@
     <div class="col-sm-7 col-md-6 col-lg-5 login">
       <form  id="login_githubLoginForm" class="form-horizontal" role="form">
         <button (click)="gitSignin()" id="login_githubLoginBtn" class="btn btn-primary btn-lg" >
-            <span class='fa fa-github'></span>
-            Sign In with Github
+            Sign In with Red Hat Developers
         </button>
```

Comment:
You can sign-in/up with GitHub via developers.redhat.com. It supports social login.